### PR TITLE
Registration form

### PR DESCRIPTION
**Pull request by itsmesatwik**
Completed form, using modals.
screen shot 2018-07-23 at 11 53 27 am
screen shot 2018-07-23 at 11 54 13 am

### CODE REVIEW

File: source/styles/main.scss [hunk 1]
```diff
@@ -9,7 +10,7 @@ body {
   width: 100vw;
   overflow-x: hidden;
   font-family: 'Roboto', sans-serif;
-  background-color: #28323f;
+  background-color: #14716c;
```
Comment (modnikhil) [inline source/styles/main.scss]: No lol

File: source/components/home/navigation/navigation.js [hunk 2]
```diff
@@ -32,13 +37,16 @@ class Navigation extends Component {
...
-                    <Nav pullRight>
+                    <Nav>
                         <NavItem onClick={() => this.scrollTo('About')} eventKey={1} className="Navigation__item--active">About</NavItem>
                         <NavItem onClick={() => this.scrollTo('Speakers')} eventKey={2} >Speakers</NavItem>
                         <NavItem onClick={() => this.scrollTo('Sponsors')} eventKey={3} >Sponsors</NavItem>
                         <NavItem onClick={() => location.href="https://blog.reflectionsprojections.org"} eventKey={4}>Blog</NavItem>
                         <NavItem onClick={() => location.href="/#/faq"} eventKey={5}>FAQ</NavItem>
                     </Nav>
+                    <Nav bsStyle="pills" pullRight>
+                        <Button onClick={() => render(<Registration/>)} bsStyle="success" bsSize="large">Register</Button>
```
Comment (modnikhil) [inline source/components/home/navigation/navigation.js]: Modal doesn't pop up when I click the button :/

Comment (itsmesatwik) [reply source/components/home/navigation/navigation.js]: now it does

File: source/styles/main.scss (hunk 1 above)
Comment (itsmesatwik) [reply source/styles/main.scss]: reverted

**Pull request closed without merging**